doc/user/simple.xml

```diff
@@ -66,6 +66,8 @@
    </para>
 
    <programlisting>
+#include <stdio.h>
```

Comment:
Change to include proper markup for < and >
```#include &lt;stdio.h&gt;```

Reply:
I will fix it soon, thanks.

I am learning scons for the use of [godot](https://github.com/godotengine/godot), and is learing scons's latest [User Guide](https://scons.org/doc/production/HTML/scons-user/index.html). Actually, the design of scons is very interesting to me, and I like the intuitive design. I will read the guide thoroughly. I might find additional typos that need to be fixed, so I reviewed the Github test failure report to learn how to run tests on my local computer and found the following commands.

```sh
# set up environment
python -m pip install --upgrade pip setuptools wheel
python -m pip install ninja psutil

# run document tests
python runtest.py test/import.py test/ninja
```
I want to know if these commands is right for me, any reply is valuable, thanks.